docs/content/get-started/upload-launch.md

```diff
@@ -82,26 +82,27 @@ The next step is to [start a cloud deployment](https://docs.improbable.io/refere
 <br/>
 To launch your cloud deployment:
 
-First enter the parameters in the Deployment Launcher section of the Deployments window. Complete the settings as below:
+1\. First enter the parameters in the **Deployment Launcher** section of the Deployments window. Complete the settings as below:
 <br/>
 
 |Setting name|The setting you add|
 |-----|------|
 |**Assembly Name**| The name you gave the assembly in step 2 - for example; `myassembly`. <br/>(The name needs to conform to the following regex: `[a-zA-Z0-9_.-]{5,64}`.)|
 |**Deployment Name**| A name of your choice - for example; `shootyshooty`. <br/>(The name needs to conform to the following regex: `[a-z0-9_]{2,32}`.)|
-|**Snapshot File**|`snapshots/cloud.snapshot`<br/>Note that this is a relative path from the root of your SpatialOS Project.|
-|**Launch Config File** | `cloud_launch_large.json`<br/>Note that this is a relative path from the root of your SpatialOS Project.|
+|**Snapshot File**|`snapshots/cloud.snapshot`<br/>Note that this is a relative path from the root of your SpatialOS project.|
+|**Launch Config File** | `cloud_launch_large.json`<br/>Note that this is a relative path from the root of your SpatialOS project.|
 |**Enable Simulated Players**| Ensure this is checked.|
-|**Override Name**| Leave this checkbox UNchecked.|
+|**Override Name**| Leave this UNchecked.|
```

Comment:
Is UN capitalised here for emphasis? I'd prefer to write it as "unchecked" unless we've got a reason for the capitalisation.